### ISSUE TITLE
getQueryVariable assumes window is available stopping ability to use editor with SSR

### ISSUE THREAD

**Issue by gust42** [locize/locize-editor#20]
## 🐛 Bug Report

When i add locize-editor to my Nextjs project it crashes on SSR compile because getQueryVariable gets called on script load
```
var defaultOptions = {
  url: 'https://www.locize.app',
  openDashboard: false,
  enabled: false,
  enableByQS: 'locize',
  toggleKeyCode: 24,
  toggleKeyModifier: 'ctrlKey',
  lngOverrideQS: 'useLng',
  lngOverride: null,
  hasNamespacePrefixed: false,
  autoOpen: true,
  onEditorSaved: function onEditorSaved(lng, ns) {},
  mode: getQueryVariable('locizeMode') || 'iframe',
  iframeContainerStyle: 'z-index: 2147480000; position: fixed; top: 0; right: 0; bottom: 0; width: 600px; box-shadow: -3px 0 5px 0 rgba(0,0,0,0.5);',
  iframeStyle: 'height: 100%; width: 600px; border: none;',
  bodyStyle: 'margin-right: 605px;'
};
``` 

if i remove getQueryVariable call in defaultOptions everthing works as it should

my proposed solution is to check if window exists in getQueryVariable and if it doesnt return false

## To Reproduce

Create a new NextJS project and import locize-editor package

## Expected behavior

Editor should not assume window is available on import of script

**Comment by adrai** [comment]
Would you like to submit a PR?

**Comment by gust42** [comment]
Sure!

**Comment by adrai** [comment]
Thank you, released with v3.0.2
